lib/git_reflow/git_helpers.rb

```diff
@@ -58,8 +58,9 @@ def merge_feature_branch(feature_branch_name, options = {})
     end
 
     def append_to_squashed_commit_message(message = '')
-      run "echo \"#{message}\" | cat - .git/SQUASH_MSG > ./tmp_squash_msg"
-      run 'mv ./tmp_squash_msg .git/SQUASH_MSG'
+      git_root_dir = `echo $(git rev-parse --show-toplevel)`.strip
```

Comment:
So, the change becomes larger than I originally expected. There are 3 points:
1. stub_command(..) doesn't work
2. run doesn't return output if loud=true

I've fixed the above points too.